csrc/executor.cpp

```diff
@@ -1891,4 +1893,267 @@ float FusionExecutor::runRtc(
   return kernel_time_ms;
 }
 
+flatbuffers::Offset<serde::FusionExecutor> FusionExecutor::serialize(
+    flatbuffers::FlatBufferBuilder& builder) const {
+  // table FusionExecutor {
```

Comment:
nitpick: maybe we can just have a comment pointing people to look at the definition in fusion_cache.fbs, rather than having a copy here.
copy is hard to maintain.